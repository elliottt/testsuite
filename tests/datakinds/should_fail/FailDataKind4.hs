{-# LANGUAGE DataKinds #-}

module FailDataKind4 where

-- this should fail, as kinds and types live in the same namespace
data kind K = T
data type K = C
