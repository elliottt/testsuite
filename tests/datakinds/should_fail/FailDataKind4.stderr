
FailDataKind4.hs:7:11:
    Multiple declarations of ‛K’
    Declared at: FailDataKind4.hs:6:11
                 FailDataKind4.hs:7:11
